lib/improver/wind_downscaling.py

```diff
@@ -734,7 +734,7 @@ def calc_av_ppgrid_res(self, a_cube):
         [exp_xname, exp_yname] = ["projection_x_coordinate",
                                   "projection_y_coordinate"]
         exp_unit = Unit("m")
-        if (x_name is not exp_xname) or (y_name is not exp_yname):
+        if (x_name != exp_xname) or (y_name != exp_yname):
```

Comment:
The previous version was entering the if loop even when the expected names are found. It seems that 'is not' checks the identities are not the same, whereas != tests for inequality between the strings. I think that because x_name comes from the name() method being called on the coordinate, it has a different 'identity' to the string exp_xname, even though the strings are equal. The problem in the original code is easy to create on the command line, for ex:
```
> test = cube.coord(axis='x')[0].name()
> print test
projection_x_coordinate
> test=='projection_x_coordinate'
True
> test is 'projection_x_coordinate'
False
```